requirements-dev.txt

```diff
@@ -4,9 +4,9 @@ Appium-Python-Client<=0.52
 flake8==3.8.3
 nose==1.3.7
 splinter>=0.13.0
--e git+https://github.com/qld-gov-au/ckanext-datarequests@2.1.1-qgov#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@3.0.1#egg=ckanext-data_qld
--e git+https://github.com/qld-gov-au/ckan-ex-qgov@3.1.0#egg=ckan-ex-qgov
+-e git+https://github.com/qld-gov-au/ckanext-datarequests@2.1.2-qgov#egg=ckanext-datarequests
```

Comment:
do you think we could get the upstream project to incorperate most of our changes? https://github.com/conwetlab/ckanext-datarequests

Reply:
There's been no activity on it in years, but we could try.